docs/crio.8.md

```diff
@@ -218,6 +219,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--enable-tracing**: Enable OpenTelemetry trace data exporting.
 
+**--evented-pleg**: Enable CRI-O to generate the container pod-level events in order to optimize the performance of the Pod Lifecycle Event Generator (PLEG) module in Kubelet.
```

Comment:
What do you think of `--emitted-events`. @haircommander thoughts on the name?

Seems weird to tie this directly with the pleg.

Reply:
I get what you mean, but I'm not as worried about it, as kubelet is the only client we care about. I think it would be most clear as `--emit-pod-and-container-events` but that's too verbose. I think pleg describes what they're for

Reply:
I think we should follow the `--enable-…` pattern for features to be used. Let me throw in a third opinion, how about: `--enable-pod-events`?

Reply:
I like `--enable-pod-events`.

Reply:
+1